src/transformers/utils/dummy_pt_objects.py

```diff
@@ -716,6 +716,13 @@ def __init__(self, *args, **kwargs):
         requires_backends(self, ["torch"])
 
 
+class AutoModelForTextEncoding(metaclass=DummyObject):
```

Comment:
There should be the equivalent `TFAutoModelForTextEncoding` in `dummy_tf_objects.py`

Reply:
Ah perfect! Thanks for flagging!